task/project/task/views.py

```diff
@@ -23,17 +25,25 @@ def post(self, request):
             if not Employee_id(assigned_id):
                 return Response(
                     unauthorised, status=status.HTTP_401_UNAUTHORIZED
-                )
+            )
             serializer.initial_data["assigned_by"] = request.user.id
             if not serializer.is_valid():
                 return Response(
                     serializer.errors, status=status.HTTP_400_BAD_REQUEST
                 )
             serializer.save()
+            employee_email = CustomUser.objects.get(id=serializer.data["assigned_to"])
+            send_mail(
```

Comment:
We may need to send emails with different subjects and body create a function for that.